### ISSUE TITLE
Declarations in For loops are ignored

### ISSUE THREAD

**Issue by genius257** [genius257/vscode-autoit#50]
```au3
Global $aItems = [1, 2, 3]
For $item In $aItems
    ConsoleWrite($item & @CRLF)
Next
```
Currently `$item` will not work for declaration lookup or completion suggestions.

**Comment by genius257** [comment]
The problem seems to be with the parser. Currently the variable declared in a `ForStatement` is parsed as a normal  `VariableIdentifier`, instead as the expected `VariableDeclaration`.

**Comment by genius257** [comment]
Fixed via 08d28d2616a56f4710dfa71da85af4579ec8daae
